pkg/metadata/scheduler.go

```diff
@@ -95,6 +100,12 @@ func (c *Scheduler) AddCollector(name string, interval time.Duration) error {
 			case healthDeadline := <-sc.healthHandle.C:
 				cancel()
 				ctx, cancel = context.WithDeadline(context.Background(), healthDeadline)
+			case <-firstRun.C:
+				log.Debugf("calling first run on '%s' metadata collector", name)
+				if err := p.Send(ctx, c.demux.Serializer()); err != nil {
+					log.Errorf("Unable to send '%s' metadata first run: %v", name, err)
+				}
+				sc.sendTimer.Reset(interval) // Reset the timer, so it fires again after `interval`.
```

Comment:
Correct me if I'm wrong, but I think we shouldn't reset a running timer (not sure what happens if we do tbh !)

According to the [doc](https://pkg.go.dev/time#Timer.Reset):
> For a Timer created with NewTimer, Reset should be invoked only on stopped or expired timers with drained channels.

The solutions I see are either to only have one timer (cf my other question) or not to reset it after the first send (it means that we send after 1m, 10m, 20m, etc instead of 1m, 11m, 21m, etc).